src/navigator_data_ingest/base/types.py

```diff
@@ -132,6 +132,7 @@ class UpdateTypes(str, Enum):
     UpdateTypes.NAME: "document_name",
     UpdateTypes.DESCRIPTION: "document_description",
     UpdateTypes.SOURCE_URL: "document_source_url",
+    UpdateTypes.PUBLICATION_TS: "publication_ts",
```

Comment:
Not good that we have two places that define `publication_ts` as a string - maybe we can write `PipelineFieldMapping` in thers of `UpdateTypes` ?